numpy/core/arrayprint.py

```diff
@@ -1192,13 +1199,16 @@ def __call__(self, x):
         else:
             return "({})".format(", ".join(str_fields))
 
+# for backwards compatibility
+StructureFormat = StructuredVoidFormat
```

Comment:
Should we add a deprecation warning for use of StructureFormat, the same way we did for LongFloatFormat?

Reply:
Sure, wouldn't hurt